docs/services/slam/cartographer/_index.md

```diff
@@ -34,318 +22,467 @@ The source code for this module is available on the [`viam-cartographer` GitHub
+   On the cartographer panel, you can start a mapping session.
+
+   When you start a mapping session:
+
+   - Viam spins up an instance of the `cartographer` module in the cloud to execute the Cartographer algorithm.
+   - Viam's [Data Capture](/services/data/configure-data-capture/) continuously monitors and captures your robot's sensor data while the robot is running.
+   - Cartographer uses the data captured from when you click **Start session** until you click **End session** to create the map.
+
+   Once you click **End session**, the map is saved to cloud storage and visible on your **Location** page under **SLAM library**.
+
+   Enter a name for your new map and click **Start session**.
+   Wait for the slam session to finish starting up in the cloud, which **takes about 2 minutes**.
+
+   Make sure to either **manually refresh**, or **change the refresh frequency** to something other than `Manual`.
+
+   ![slam RC card start session](/services/slam/slam-RC-card-start-session.png)
+
+   While the slam session is starting, you will see a loading screen.
+
+   ![slam RC card wait for session to finish starting](/services/slam/slam-RC-card-wait-for-session-to-finish-starting.png)
+
+   Once the slam session has finished starting, your first pointcloud will appear.
+
+   ![slam RC card first pointcloud](/services/slam/slam-RC-card-first-pointcloud.png)
+
+   You can see that your cloud slam session is in progress from your **Location** page's **SLAM library** tab.
+
+   ![offline mapping maps computing table](/services/slam/offline-mapping-maps-computing-table.png)
+
+   When you are ready to end the slam session, return to your robot's **Control** tab and click **End session**.
+   If you do not click **End session**, the slam session will automatically end after 45 minutes.
+
+   Once the session has ended, the map is saved to your **Location** page's **SLAM library** tab.
+
+   ![offline mapping available maps](/services/slam/offline-mapping-available-maps.png)
+
+   You can click `View map` to view the map in a dynamic pointcloud viewer.
+
+   ![slam library view map](/services/slam/slam-library-view-map.png)
+
+### Update an existing map
+
+To update an existing map with new pointcloud data from a new SLAM session, follow the instructions below.
+Updating an existing map uses an instance of the `cartographer` module running in the cloud, and _does not_ overwrite the existing map.
+You must have first [created a new map](#create-a-new-map) to be able to update an existing map.
+
+1. Configure your `cartographer` SLAM service
+
+   {{< tabs name="Update existing map">}}
+   {{% tab name="Config Builder" %}}
+
+   - First, follow the configuration steps in [Create a new map](#create-a-new-map).
+
+   - Then, configure **Select map** and **Map version** with the name and version of the map you would like to update.
```

Comment:
[optional] It's a little odd to say "Then, configure 'Select map' and 'Map version'" because in the UI, the 'Select map' and 'Map version' boxes come first:

<img width="782" alt="Screenshot 2023-10-20 at 12 05 24 PM" src="https://github.com/viamrobotics/docs/assets/1517691/8eb48e6a-79da-421e-b486-84d434736de8">

That's why I had phrased it as:

> The configuration is similar to the configuration for [creating a new map](https://docs.viam.com/services/slam/cartographer/#create-new-map), except you additionally configure the following Attribute:

> * "Select map", "Map version": Provide the name and version of the map you would like to update. You can see more details about the available maps from this robot’s Location page under the SLAM library tab.

I think people will understand what to do though, and it's okay to leave it this way as well